src/wp-includes/rest-api/endpoints/class-wp-rest-font-families-controller.php

```diff
@@ -0,0 +1,518 @@
+<?php
+
+/**
+ * Rest Font Families Controller.
+ *
+ * This file contains the class for the REST API Font Families Controller.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+/**
+ * Font Families Controller class.
+ *
+ * @since 6.4.0
+ */
+class WP_REST_Font_Families_Controller extends WP_REST_Controller {
```

Comment:
@matiasbenedetto Sorry if I am missing this, but why is this a custom controller? Doesn't this map to the post contoller for font CPT?